deploy/helm/sifnode/templates/ebrelayer-deployment.yaml

```diff
@@ -65,8 +65,6 @@ spec:
             mountPath: /root
       volumes:
       - name: root
```

Comment:
@intl-man if we want to remove the claim we need to remove the name of the claim volume as well.